wagtail/admin/templates/wagtailadmin/account/password_reset/confirm.html

```diff
@@ -22,15 +22,17 @@ <h1>{% trans "Set your new password" %}</h1>
                 <button type="submit" class="button">{% trans 'Reset password' %}</button>
             </form>
         {% else %}
-            <h1>{% trans "Invalid password reset link" %}</h1>
-            <div class="messages">
-                <ul>
-                    <li class="error">
-                        {% trans "The password reset link was invalid, possibly because it has already been used." %}
-                    </li>
-                </ul>
-            </div>
-            <a href="{% url 'wagtailadmin_password_reset' %}" class="button">{% trans 'Request a new password reset' %}</a>
+            <form class="content" method="post" novalidate>
+                <h1>{% trans "Invalid password reset link" %}</h1>
```

Comment:
I think this form is not needed.  Maybe an outer wrapper div.content for the while thing would be better?